components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.cpp

```diff
@@ -0,0 +1,117 @@
+#include "EmbeddedRemeshing.hpp"
+
+namespace wmtk::components::internal {
+EmbeddedRemeshing2D::EmbeddedRemeshing2D(
+    Eigen::MatrixXi& E_,
+    Eigen::MatrixXd& V_,
+    double blank_rate_,
+    double resolute_area_)
+    : E(E_)
+    , V(V_)
+    , blank_rate(blank_rate_)
+    , resolute_area(resolute_area_)
+{
+    F.resize(0, 0);
+    for (int i = 0; i < V.rows(); ++i) {
```

Comment:
Minor thing but please either use `size_t` or `long` here. We are avoiding `int` in the toolkit. In this case here, I think `size_t` is what you actually want.